ci-operator/step-registry/cco/remove-root-secret/cco-remove-root-secret-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+function remove_root_secret()
+{
+    local secret_name=$1
+    echo "Removing secret ${secret_name} from kube-system"
+    oc delete secret -n kube-system ${secret_name}
+    echo "Removed."
+
+}
+
+mode=$(oc get cloudcredentials cluster -o=jsonpath="{.spec.credentialsMode}")
+
+case "${CLUSTER_TYPE:-}" in
+aws*(*))
+    echo "Cluster "
+    if [[ ${mode} == "" ]] || [[ ${mode} == "Mint" ]]; then
+        remove_root_secret "aws-creds"
+    else
+        echo "Removing root secret in credentialsMode \"${mode}\" on AWS is not supported. Skip and exit now."
+        exit 0
+    fi
+    ;;
+gcp)
+    # TODO: check default credentialsMode on GCP
+    # remove_root_secret "aws-creds"
+    exit 0
+    ;;
+*)
+    echo "Cluster type '${CLUSTER_TYPE}' is not supported, Skip and exit now."
+    exit 0
```

Comment:
I suggest to `exit 1` for more restrict checking when it is being refereced.

Reply:
Updated.